x-pack/plugins/enterprise_search/common/types/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IAccount as IAppSearchAccount } from './app_search';
+import { IAccount as IWorkplaceSearchAccount, IOrganization } from './workplace_search';
+
+export interface IInitialAppData {
+  readOnlyMode?: boolean;
+  ilmEnabled?: boolean;
+  configuredLimits?: IConfiguredLimits;
```

Comment:
Why is this top-level and not nested under `appSearch`?

Reply:
I believe Oleksiy mentioned that Workplace Search either uses configurable limits as well (possibly in the back-end) or may, in the future, add more configurable limits to the front-end.